### ISSUE TITLE
Install fails on windows (8.1)

### ISSUE THREAD

**Issue by frelun** [dachev/node-cld#17]
Since 2.3.2 I get the following errors when installing cld (running windows 8.1 x64):
```
> cld@2.4.0 postinstall C:\tmp\node_modules\cld
> rm -r ./deps/ && find ./build -type f | grep -v 'cld.node' | xargs -I{} rm {}

'rm' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "cld"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! cld@2.4.0 postinstall: `rm -r ./deps/ && find ./build -type f | grep -v 'cld.node' | xargs -I{} rm {}`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cld@2.4.0 postinstall script 'rm -r ./deps/ && find ./build -type f | grep -v 'cld.node' | xargs
-I{} rm {}'.
npm ERR! This is most likely a problem with the cld package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm -r ./deps/ && find ./build -type f | grep -v 'cld.node' | xargs -I{} rm {}
npm ERR! You can get their info via:
npm ERR!     npm owner ls cld
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\tmp\npm-debug.log
```
No problems installing cld@2.3.1 however

**Comment by dachev** [comment]
@frelun Thanks for the heads up. I'll fix it on the weekend.

**Comment by dachev** [comment]
@frelun I released 2.4.2. It should resolve the issue. Can you give it try and confirm?

**Comment by frelun** [comment]
Works fine now, Thanks!
